src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -201,8 +201,8 @@ export default class TutorialView {
                 if (prop.aircraft.list.length <= 0) {
                     return t;
                 }
-
-                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+                const reverseList = [...prop.aircraft.list].reverse();
```

Comment:
@n8rzz oh, I guess you're saying `.reverse()` mutates. I see what's going on. That's a clever solution then!

@Dabea The list of aircraft in the AircraftController still has new aircraft added to the end, so the `[0]` index of the filter restults will be the "first" arrival or departure, like we want. The aircraft we want _is_ the one whose strip is at the bottom of all other aircraft of their category, so it should get the right one without any reversing.